app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,34 @@
 class SessionsController < ApplicationController
   def create
+    user_info = request.env['omniauth.auth']
+    name = user_info[:info][:name]
+    uid = user_info[:uid]
+    email = user_info[:info][:email]
+    token = user_info[:credentials][:token]
+    
+    #found_user = User.find_or_create_by(uid: uid, email: email, token: token, name: name)
+
+    found_user = User.find_or_create_by(name: name) do |user|
+      user.uid = uid
+      user.email = email
+      user.token = token
+      user.name = name
+    end
+
+    found_user.save
+    session[:uid] = uid
+    redirect_to dashboard_path
   end
```

Comment:
I believe this is the point at which we would also send a post request to the BE app to create the user there

Reply:
Just kidding, we'd have to it before this if we're going to send it only if the user doesn't already exist in the FE database. I guess the alternative would be to _always_ send the post request and let the backend sort out if they have the user or not, but it seems redundant and could possibly open the door to errors in storing or duplicating records on the back end